metpy/plots/declarative.py

```diff
@@ -670,15 +723,66 @@ class Plot2D(HasTraits):
     _need_redraw = Bool(default_value=True)
 
     field = Unicode()
+    field.__doc__ = """Name of the field to be plotted.
+
+    This is the name of the variable from the dataset that is to be plotted. An example,
+    from a model grid file that uses the THREDDS convention for naming would be
+    `Geopotential_height_isobaric` or `Temperature_isobaric`. For GOES-16/17 satellite data it
+    might be `Sectorized_CMI`. To check for the variables available within a dataset, list the
+    variables with the following command assuming the dataset was read using xarray as `ds`,
+    `list(ds)`
+    """
+
     level = Union([Int(allow_none=True, default_value=None), Instance(units.Quantity)])
+    level.__doc__ = """The level of the field to be plotted.
+
+    This is a value with units to choose the desired plot level. For example, selecting the
+    850-hPa level, set this parameter to ``850 * units.hPa``
+    """
+
     time = Instance(datetime, allow_none=True)
+    time.__doc__ = """Set the valid time to be plotted as a datetime object.
+
+    If a forecast hour is to be plotted the time should be set to the valid future time, which
+    can be done using the ``datetime`` and ``timedelta`` modules from base Python.
```

Comment:
Can get it to link to Python docs with:
```suggestion
    can be done using the `~datetime.datetime` and `~datetime.timedelta` objects from the
    Python standard library.
```

This only works with the single `` ` ``, and the `~` makes it elide away the fully-qualified name.

Reply:
Neat. Done.